src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -941,6 +942,20 @@ public void verifyFieldValue (String fieldId, String expectedValue) {
         assertEquals(expectedValue,
                 browser.driver.findElement(By.id(fieldId)).getAttribute("value"));
     }
+    
+    /**
+     * This method verifies that the page source does not contain the given
+     * searchString.
+     * 
+     * @param searchString
+     *            the substring that we want omitted from the page source
```

Comment:
`want to omit`? And let's see if we can make it inline with `@param searchString` or not.